### ISSUE TITLE
Users are clicking "Accept" webcam settings BEFORE its ready

### ISSUE THREAD

**Issue by steveseguin** [steveseguin/obsninja#45]
I need to make it harder to press the "ready" button, perhaps by activiating it only after the stream is registered. pressing the button too early causes it to fail.

On some older computers with odd cameras, this can take a few seconds even, so the user may rush to press a button and break things

**Comment by steveseguin** [comment]
I fixed a bit, by giving a verbal "wait" message.  It won't fix the problem completely, but it should at least limit the issue.
